_data/apps.yml

```diff
@@ -1101,3 +1101,14 @@
     - "database"
     - "design"
   icon: "collectie.png"
+-
+  name: "Calculator"
+  description: "A Simple Calculator with Google UI"
+  website: "https://softvilla.com.pk"
```

Comment:
When I try to visit the website I see the following warning:

<img width="698" alt="screen shot 2016-05-28 at 10 17 36 am" src="https://cloud.githubusercontent.com/assets/671378/15628732/722b0444-24bd-11e6-82a5-ab4097a6bcc2.png">